unified-scheduler-pool/src/lib.rs

```diff
@@ -231,6 +232,22 @@ impl HandlerContext {
     fn banking_stage_helper(&self) -> &BankingStageHelper {
         self.banking_stage_helper.as_ref().unwrap()
     }
+
+    fn clone_for_scheduler_thread(&self) -> Self {
+        let mut context = self.clone();
+        if self.banking_stage_helper.is_some() {
+            context.disable_banking_packet_handler();
+        }
+        context
+    }
+
+    fn disable_banking_packet_handler(&mut self) {
+        self.banking_packet_receiver = never();
+        self.banking_packet_handler = Box::new(|_, _| {
+            // This is safe because of the paired use of never() just above.
+            unreachable!()
```

Comment:
nit: just put the comment in the `unreachable!`:

```suggestion
            unreachable!("paired with never() receiver, this cannot be called")
```

or something like that.